assets/stylesheet/style.css

```diff
@@ -0,0 +1,144 @@
+body {
+    background: #fff;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.body-wrap {
+    background-color: #fff;
+    margin-top: 8em;
+}
+
+.fixed-top {
+    margin-top: 0;
+}
+
+.header-one {
+    background-color: #F72210;
+    height: 50px;
+    width: 100%;
+}
+
+.nav-item {
+    margin-right: 45px;
+}
+
+
+/*
+.sign_in_subscribe>button {
+    font-size: .8rem;
+    border-radius: 0;
+}
+
+.sign_in_subscribe>a {
+    font-size: .6rem;
+}
+*/
+
+.exhibit {
+    background-color: white;
+    margin-left: 70%;
+    padding: 1% 5% 50% 3%;
+}
+
+footer {
+    width: 100%;
+}
+
+.fa-bars {
+    color: white;
+}
+
+.left_main {
+    width: 60%;
+}
+
+.ryt_main {
+    width: 100%;
+}
+
+.h1 {
+    font-size: .9rem;
+    text-align: center;
+    font-weight: bolder;
+    color: #000;
+    font-family: inherit;
+}
+
+
+/*
+@media only screen and (max-width: 480px) {
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.